src/main/scala/edu/berkeley/ce/rockslicing/NumericUtils.scala

```diff
@@ -5,7 +5,7 @@ import scala.math.BigDecimal
  * Numerical utilities for implementation and test cases.
  */
 object NumericUtils {
-  val EPSILON = 1.0e-6
+  val EPSILON = 1.0e-5
```

Comment:
The comparison between faces sometimes is not matching up to the 6th decimal place, even though they should be equal. Should I instead modify this within `approximateEquals` in `Block.scala`?